SeleniumFramework/DriverCore/WebDriverAction.cs

```diff
@@ -1,14 +1,10 @@
 ﻿using NUnit.Framework;
 using OpenQA.Selenium;
+using OpenQA.Selenium.Interactions;
 using OpenQA.Selenium.Support.UI;
 using SeleniumFramework.Reporter;
 using System;
 using System.Collections.Generic;
-using System.IO;
-using System.Linq;
-using System.Text;
-using System.Threading.Tasks;
-
 
 namespace SeleniumFramework.DriverCore
```

Comment:
Please handle everything by xpath, it will be confusing when also using ID. You duplicated some codes for handling that.